recipes/ephemeral.rb

```diff
@@ -0,0 +1,18 @@
+# This is only intended to be run on ephemeral, guarded instances
+# such as test-kitchen, to make things easier to set up.
+
+# Include Chef Sugar here for `windows?`
+include_recipe 'chef-sugar::default'
+require 'chef/sugar/core_extensions'
+
+if windows?
+  powershell_script 'Disable password complexity requirements' do
+    code <<-EOH
+      secedit /export /cfg $env:temp/export.cfg
+      ((get-content $env:temp/export.cfg) -replace ('PasswordComplexity = 1', 'PasswordComplexity = 0')) | Out-File $env:temp/export.cfg
+      secedit /configure /db $env:windir/security/new.sdb /cfg $env:temp/export.cfg /areas SECURITYPOLICY
+    EOH
+  end
+end
```

Comment:
We can just integrate this into `omnbius::_user` and place the following guard on the `powershell_script` resource:

```ruby
only_if { windows? && vagrant? }
```